pipelines/ops/druid/Jenkinsfile.ingestion

```diff
@@ -25,8 +25,22 @@ node("ops-slave") {
                 module = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-2].trim()
                 jobName = sh(returnStdout: true, script: "echo $JOB_NAME").split('/')[-1].trim()
                 currentWs = sh(returnStdout: true, script: 'pwd').trim()
+        // populating the vars for raw-overlord and rollup-overlord cluster respectively.
+                list_vars="$ingestion_task_names";
+                println(list_vars);
+                List<String> list_vars_array = list_vars.split(',');
+                List<String> rollup_vars = [], raw_vars = [];
+                list_vars_array.each {
+                    if(it.startsWith("rollup")) {
+                        rollup_vars.add(it);
+                    }
+                    else
+                       raw_vars.add(it);
+                }
+                rollup_ingestion_task_names = rollup_vars.join(",");
+                raw_ingestion_task_names = raw_vars.join(",");
                 ansiblePlaybook = "${currentWs}/ansible/druid-ingestion.yml"
-                ansibleExtraArgs = "--extra-vars \"ingestion_task_names=${params.ingestion_task_names} cluster=${params.cluster}\" --tags ${params.action} --vault-password-file /var/lib/jenkins/secrets/vault-pass -vv"
+                ansibleExtraArgs = "--extra-vars \"rollup_ingestion_task_names=$rollup_ingestion_task_names raw_ingestion_task_names=$raw_ingestion_task_names\" --tags populate_var,${params.action} --vault-password-file /var/lib/jenkins/secrets/vault-pass -vv"
```

Comment:
Directly pass the original variables `rollup_vars.join(",")` and `raw_vars.join(",")` in `--extra-vars`

Reply:
yeah it worked. The curly braces did a trick. Earlier I tried with out curly braces did not work. 

Updated as suggested.